metricbeat/module/windows/perfmon/_meta/fields.yml

```diff
@@ -0,0 +1,26 @@
+- name: perfmon
```

Comment:
For dynamic metricsets, this will change so what we nomrally do is just define windows (module) part and leave the rest empty. Check the jmx metricset as an example.